rental.rb

```diff
@@ -0,0 +1,14 @@
+class Rental
+  attr_accessor :date
+  attr_reader :book, :person
```

Comment:
- [ ] Please make sure that the `person` and `book` are set with the `attr accessor` property. If not, you will only be able to read the values of a `person` and a `book` from a "Rental" instance; you will not be able to use assignment to directly alter these values. For instance, you could use `rental.book` to retrieve the associated book object, but you couldn't use `rental.book = new book` to replace the rental's associated book. 👌😊

```suggestion
   attr_accessor :date, :book, :person
```

Reply:
@ridwanediallo, Thank you for the detailed explanation. I appreciate it.😁